kbs/docs/config.md

```diff
@@ -144,13 +144,14 @@ attestation. The following properties can be set.
 
 >gRPC CoCo AS is available only when `coco-as-grpc` feature is enabled.
 
-| Property                 | Type    | Description                                                                              | Required | Default |
-|--------------------------|---------|------------------------------------------------------------------------------------------|----------|---------|
-| `timeout`                | Integer | The maximum time (in minutes) between RCAR handshake's `auth` and `attest` requests      | No       | 5       |
-| `base_url`               | String  | Intel Trust Authority API URL.                                                           | Yes      | -       |
-| `api_key`                | String  | Intel Trust Authority API key.                                                           | Yes      | -       |
-| `certs_file`             | String  | URL to an Intel Trust Authority portal or path to JWKS file used for token verification. | Yes      | -       |
-| `allow_unmatched_policy` | Boolean | Determines whether to ignore the `policy_ids_unmatched` token claim.                     | No       | false   |
+| Property                 | Type         | Description                                                                              | Required | Default |
+|--------------------------|--------------|------------------------------------------------------------------------------------------|----------|---------|
+| `timeout`                | Integer      | The maximum time (in minutes) between RCAR handshake's `auth` and `attest` requests      | No       | 5       |
+| `base_url`               | String       | Intel Trust Authority API URL.                                                           | Yes      | -       |
+| `api_key`                | String       | Intel Trust Authority API key.                                                           | Yes      | -       |
+| `certs_file`             | String       | URL to an Intel Trust Authority portal or path to JWKS file used for token verification. | Yes      | -       |
+| `allow_unmatched_policy` | Boolean      | Determines whether to ignore the `policy_ids_unmatched` token claim.                     | No       | false   |
+| `policy_ids`             | String array | List of one or more quoted and comma-separated policy IDs.                               | No       | `[]`    |
```

Comment:
It looks like these are only used for ITA at least in the current implementation so maybe the scope of these configs should be ITA?

Reply:
That is already covered: we have sections for AS specific config clags: "When `type` is set to `intel_ta`, KBS will try to connect a remote Intel TA service for attestation. The following properties can be set...."